index.html

```diff
@@ -0,0 +1,242 @@
+          </ul>
+        </div>
+        <button type="button" class="big-button">See project</button>
+      </div>
+
+      <!---work section card 2-->
+      <div class="workd-card">
+        <img src="images/SnapshootPortfolio2.png" alt="" />
+        <h1>Multi-Post Stories</h1>
+        <!-- <div class="list-container-1"> -->
+        <ul class="list-container-1">
+          <li class="canopy">canopy</li>
+
+          <li>
+            <img src="images/Counter.png" alt="" />
+            Back End Dev
+          </li>
+
+          <li>
+            <img src="images/Counter.png" alt="" />
+            2015
+          </li>
+        </ul>
+        <!-- </div> -->
+
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="list-container-2">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <button type="button" class="big-button">See project</button>
+      </div>
+
+      <!---work section card 3-->
+      <div class="workd-card">
+        <img src="images/SnapshootPortfolio3.png" alt="" />
+        <h1>Tonic</h1>
+        <!-- <div class="list-container-1"> -->
+        <ul class="list-container-1">
+          <li class="canopy">canopy</li>
+
+          <li>
+            <img src="images/Counter.png" alt="" />
+            Back End Dev
+          </li>
+
+          <li>
+            <img src="images/Counter.png" alt="" />
+            2015
+          </li>
+        </ul>
+        <!-- </div> -->
+
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="list-container-2">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <button type="button" class="big-button">See project</button>
+      </div>
+      <!---work section card 4-->
+      <div class="workd-card">
+        <img src="images/SnapshootPortfolio4.png" alt="" />
+        <h1>Multi-Post Stories</h1>
+
+        <ul class="list-container-1">
+          <li class="canopy">canopy</li>
+
+          <li>
+            <img src="images/Counter.png" alt="" />
+            Back End Dev
+          </li>
+
+          <li>
+            <img src="images/Counter.png" alt="" />
+            2015
+          </li>
+        </ul>
+        <!-- </div> -->
+
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="list-container-2">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <button type="button" class="big-button">See project</button>
+      </div>
+      <!---about my self-->
+      <div class="workd-card about-myself">
```

Comment:
- [ ] Kindly make this `section` tag instead of ordinary `div` for semantic reasons.